website/src/css/custom.css

```diff
@@ -4,27 +4,44 @@
  * work well for content-centric websites.
  */
 
+/* imported fonts */
+@import url("https://fonts.googleapis.com/css2?family=Share+Tech+Mono&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=DM+Mono&display=swap");
```

Comment:
Let's download font locally.

Reply:
Is there any reason for downloading fonts?